CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributor Guide
+
+## Environment setup and workflow
+- Clone the repo
+- Create a fresh python environemt with `.binder/requirements.txt` file
```

Comment:
@bsipocz I noticed you recently moved requirements.txt to `.binder/` from `tutorials/`. Can we keep it in `tutorials/` and create a symlink to it in `.binder/`?

Reply:
Symlinks notoriously won't work on windows, at least I had issues with keeping symlinks in repos before (and we only symlinked licences and contributing guides rather than actually important content)

Reply:
I thought users wouldn't need to touch this requirements.txt file all since notebooks in this repo should always `pip install` what they need in the first code cell -- I thought this file was only here for the rendering? I do see that the instructions below have some dependency on in it (i.e., assuming jupytext extension is installed). Since my workflow with notebooks is so different I don't know exactly what the user will need, so I'm fine with whatever's easiest (either telling them to install that extension separately or just pointing them to the requirements.txt file even though it isn't intended for them). Although, the other repo doesn't have a requirements.txt, so if this file gets moved there that's another factor to think about.